src/helpers/tests/measureTitleHelper.spec.ts

```diff
@@ -205,4 +205,64 @@ describe("measureTitleHelper", () => {
             ).toBeUndefined();
         });
     });
+
+    describe("ignoreTitles", () => {
+        function getTitleOfMeasure(
+            visualizationObject: IVisualizationObjectContent,
+            localIdentifier: string,
+        ): string {
+            const measureBucketItems = visualizationObject.buckets[0].items;
+            const matchingMeasure: IMeasure = measureBucketItems
+                .map(bucketItem => bucketItem as IMeasure)
+                .find(bucketItem => bucketItem.measure.localIdentifier === localIdentifier);
+
+            return matchingMeasure === undefined ? undefined : matchingMeasure.measure.title;
+        }
+
+        function getAliasOfMeasure(
+            visualizationObject: IVisualizationObjectContent,
+            localIdentifier: string,
+        ): string {
+            const measureBucketItems = visualizationObject.buckets[0].items;
+            const matchingMeasure: IMeasure = measureBucketItems
+                .map(bucketItem => bucketItem as IMeasure)
+                .find(bucketItem => bucketItem.measure.localIdentifier === localIdentifier);
+
+            return matchingMeasure === undefined ? undefined : matchingMeasure.measure.alias;
+        }
+
+        it.each([
+            "arithmetic_measure_created_from_complicated_arithmetic_measures",
+            "m1",
+            "m2",
+            "m3",
+            "m4",
+            "m1_pop",
+            "m1_previous_period",
+            "m1_pop_renamed",
+            "m1_previous_period_renamed",
+            "derived_measure_from_arithmetic_measure",
+            "arithmetic_measure_created_from_simple_measures",
+            "arithmetic_measure_created_from_renamed_simple_measures",
+            "arithmetic_measure_created_from_simple_measures",
+            "arithmetic_measure_created_from_arithmetic_measures",
+            "arithmetic_measure_created_from_renamed_derived_measures",
+            "invalid_arithmetic_measure_with_missing_dependency",
+            "invalid_arithmetic_measure_with_cyclic_dependency_1",
+            "invalid_arithmetic_measure_with_cyclic_dependency_2",
+        ])("should delete all measures", (value: string) => {
```

Comment:
These tests have the same name, that's not ok :)

Reply:
Fixed!